### PR TITLE
oathkeeper: add missing "token_from->cookie" documentation for jwt authenticator (#243)

### PR DESCRIPTION
**Pull request by ngrigoriev**
oathkeeper: add missing "token_from->cookie" documentation for jwt authenticator (#243)

Signed-off-by: Grigoriev, Nikolai <ngrigoriev@gmail.com>

### CODE REVIEW

**Comment by ngrigoriev** [comment]
I have just signed the CLA

**Comment by aeneasr** [comment]
Thank you - I believe that's a problem with the git username as it appears to be your full real name but GitHub expects your git user name to match your GitHub username (`ngrigoriev`). But it's obvious that it's the same person :)

File: schemas/oathkeeper.config.schema.json [hunk 1]
```diff
@@ -375,7 +375,7 @@
         },
```
Comment (aeneasr) [inline schemas/oathkeeper.config.schema.json]: Changes in this file must actually come from ORY Oathkeeper, [which you have done](https://github.com/ory/oathkeeper/pull/331/files#diff-36449d2bff0518bdb98b6dda1dbce16d). However, changes here appear to have more lines changed (several descriptions) than the file at ORY Oathkeeper. Could you maybe update the PR there and revert the changes here? I will then merge the ORY Oathkeeper PR first to ensure that the file is up to date!

Comment (ngrigoriev) [reply schemas/oathkeeper.config.schema.json]: About to verify the schema file in Oathkeeper PR to be sure

**Comment by aeneasr** [comment]
Thank you!

**Pull request merged**